app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsSubMenuAboutRobot.kt

```diff
@@ -79,32 +95,111 @@ private fun assertProductCompany() {
 
 private fun assertCurrentTimestamp() {
     onView(withId(R.id.build_date))
-        .check(matches(withText(org.mozilla.fenix.BuildConfig.BUILD_DATE)))
+        .check(BuildDateAssertion.isDisplayedDateAccurate())
 }
 
 private fun assertWhatIsNewInFirefoxPreview() {
     onView(withText("What’s new in Firefox Preview"))
         .check(matches(withEffectiveVisibility(Visibility.VISIBLE)))
+        .perform(click())
+
+    mDevice.waitNotNull(
+        Until.findObjects(By.res("org.mozilla.fenix.debug:id/mozac_browser_toolbar_title_view")),
+        TestAssetHelper.waitingTime
+    )
+
+    onView(withId(R.id.mozac_browser_toolbar_title_view)).check(
+        matches(
+            withText(
+                containsString("What's new in Firefox Preview")
+            )
+        )
+    )
+    Espresso.pressBack()
 }
 
 private fun assertSupport() {
     onView(withText("Support"))
         .check(matches(withEffectiveVisibility(Visibility.VISIBLE)))
+        .perform(click())
+
+    mDevice.waitNotNull(
+        Until.findObjects(By.res("org.mozilla.fenix.debug:id/mozac_browser_toolbar_title_view")),
+        TestAssetHelper.waitingTime
+    )
+
+    onView(withId(R.id.mozac_browser_toolbar_title_view)).check(
+        matches(
+            withText(
+                containsString("Firefox Preview | Mozilla Support")
+            )
+
+        )
+    )
+    Espresso.pressBack()
 }
 
 private fun assertPrivacyNotice() {
     onView(withText("Privacy notice"))
         .check(matches(withEffectiveVisibility(Visibility.VISIBLE)))
+        .perform(click())
+
+    mDevice.waitNotNull(
```

Comment:
this should prob go into a helper that you can pass in necessary params for verification